aspnetcore/mvc/views/tag-helpers/built-in/index.md

```diff
@@ -12,7 +12,12 @@ uid: mvc/views/tag-helpers/builtin-th/Index
 
 By [Peter Kellner](http://peterkellner.net)
 
-ASP.NET Core includes built-in Tag Helpers to boost your productivity. This section provides a list of the built-in Tag Helpers.
+Tag Helpers:
```

Comment:
* Anyone coming here likely knows what Tag Helpers are already.
* If they don't know, these bullet points don't help, they just tout advantages without explaining what TH are.

Consider a simpler approach:

```
This article provides an overview of built-in [Tag Helpers](xref:mvc/views/tag-helpers/intro).
```
Those who know what TH are just want to see  the list anyway; those who don't know aren't ready for this article and can click the link  for the intro.

Reply:
It's not an "overview" tho. 

You were going to kill link-list topics anyway, right? I'll remove the lines for now, but would you be open to just killing the whole file and have the "built-in" TOC link just open the node for built-in THs?

Reply:
btw ... how does it go? ...

```
#### [Built-in Tag Helpers](#)
```

Reply:
Yes, good point about list vs. overview. Even if we make an  expand/collapse ToC node, we need an index.md file but I'm fine with you doing that to the ToC if you want.

Reply:
How does it work tho? Leave the *index.md* file ... but what does the TOC link look like such that clicking the link only opens the node. Like this? ...

```
#### [Built-in Tag Helpers](#)
##### [Anchor Tag Helper](xref:mvc/views/tag-helpers/builtin-th/anchor-tag-helper)
##### [Cache Tag Helper](xref:mvc/views/tag-helpers/builtin-th/cache-tag-helper)
...
```

Reply:
On the kill link list point, yes but I think this is a special case of link list -- it seems useful to have a page that lists Tag Helpers.

Reply:
Just don't make the node a link at all

Reply:
Oh ... I see. I thought it might need to be a link to make some JS work to spring the node open. Ok ... cool. I'll do that on the next commit.